Dockerfile-kubepkg-rpm

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM golang:1.18.5-buster AS builder
+FROM golang:1.19-buster AS builder
```

Comment:
should we update to use `bullseye`?

Reply:
Yeah let's do that in one step.

Reply:
Do you mean in a follow up? or in this PR as well?

Reply:
In this PR here